spec/requests/api/v1/countries/update_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe 'PUT /api/v1/countries/:id', type: :request do
+  subject { put api_v1_country_path(country_id), headers: headers, params: params, as: :json }
+
+  let(:headers) { auth_headers }
+  let(:user) { create(:user) }
+  let!(:country) { create(:country) }
+  let(:country_id) { country.id }
+  let(:params) do
+    {
+      country: {
+        name: name
+      }
+    }
+  end
+
+  context 'when response is successful' do
+    let(:name) { 'foo' }
+
+    it_behaves_like 'a successful request'
+
+    it 'returns country info' do
+      subject
+
+      expect(json).to include_json(
+        id: country.id,
+        name: name
+      )
+    end
+
+    it 'does not modify country table size' do
+      expect { subject }.to change { Country.count }.by(0)
+    end
```

Comment:
following this case, you could also test attr value changes

```rb
      expect { subject }.to change { country.reload.name }.to(name)
```
it possible to add the `.from(previous_value)` method too

Reply:
Updated.